queries/nickel/indents.scm

```diff
@@ -0,0 +1,8 @@
+[
+  (term)
```

Comment:
Does this mean `(term)` consists of both `[...]` and `{...}` sections

Reply:
> Does this mean `(term)` consists of both `[...]` and `{...}` sections

@lucario387 `(term)` is the container of all value types,  including `[...]` and `{...}`